bundles/com.espressif.idf.launch.serial.ui/src/com/espressif/idf/launch/serial/ui/internal/CMakeMainTab2.java

```diff
@@ -51,9 +67,12 @@
 import com.espressif.idf.launch.serial.SerialFlashLaunchTargetProvider;
 import com.espressif.idf.launch.serial.internal.SerialFlashLaunchConfigDelegate;
 import com.espressif.idf.launch.serial.util.ESPFlashUtil;
+import com.espressif.idf.ui.EclipseUtil;
 
+@SuppressWarnings("restriction")
 public class CMakeMainTab2 extends GenericMainTab {
 	private static final String EMPTY_CONFIG_OPTIONS = "-s ${openocd_path}/share/openocd/scripts"; //$NON-NLS-1$
+	private static final String EMPTY = ""; //$NON-NLS-1$
```

Comment:
StringUtil.EMPTY 
can be used here from our own StringUtil class

Reply:
I updated it myself

Reply:
Hi @alirana01. Thanks